zkevm-circuits/src/copy_circuit.rs

```diff
@@ -553,7 +588,7 @@ impl<F: Field> CopyCircuitConfig<F> {
             || format!("assign src_addr_end {}", offset),
             self.copy_table.src_addr_end,
             offset,
-            || Value::known(F::zero()),
+            || Value::known(F::one()),
```

Comment:
I was confusing in the beginning to see this updated to `1`, then I realized it's required for passing `is_pad` constraint `is_pad == 1 - (src_addr < src_addr_end) for read row`.

I think setting `is_pad` to be `(offset % 2 == 0)` directly makes it easier to follow for me, but it's fine to keep it unchanged (since it doesn't affect performance).